docs/index.md

```diff
@@ -60,34 +65,158 @@ resource "ocm_group_membership" "my_admin" {
+      support_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Support-Role",
+      operator_iam_roles = [
+        {
+          name =  "cloud-credential-operator-iam-ro-creds",
+          namespace = "openshift-cloud-credential-operator",
+          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-cloud-credential-operator-cloud-c",
+        },
+        {
+          name =  "installer-cloud-credentials",
+          namespace = "openshift-image-registry",
+          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-image-registry-installer-cloud-cr",
+        },
+        {
+          name =  "cloud-credentials",
+          namespace = "openshift-ingress-operator",
+          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-ingress-operator-cloud-credential",
+        },
+        {
+          name =  "ebs-cloud-credentials",
+          namespace = "openshift-cluster-csi-drivers",
+          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-cluster-csi-drivers-ebs-cloud-cre",
+        },
+        {
+          name =  "cloud-credentials",
+          namespace = "openshift-cloud-network-config-controller",
+          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-cloud-network-config-controller-c",
+        },
+        {
+          name =  "aws-cloud-credentials",
+          namespace = "openshift-machine-api",
+          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-machine-api-aws-cloud-credentials",
+        },
+      ]
+      instance_iam_roles = {
+        master_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-ControlPlane-Role",
+        worker_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Worker-Role"
+      },    
+  }
+}
+resource "ocm_cluster" "rosa_cluster" {
+  name           = var.cluster_name
+  cloud_provider = "aws"
+  cloud_region   = "us-east-2"
+  product        = "rosa"
+  aws_account_id     = data.aws_caller_identity.current.account_id
+  aws_subnet_ids = var.rosa_subnet_ids
+  machine_cidr = var.rosa_vpc_cidr
+  multi_az = true
+  aws_private_link = true
+  availability_zones = ["us-east-2a", "us-east-2b", "us-east-2c"]
+  proxy = {
+    http_proxy = var.proxy
+    https_proxy = var.proxy
+  }
+  properties = {
+    rosa_creator_arn = data.aws_caller_identity.current.arn
+  }
+  wait = false
+  sts = local.sts_roles
+}
+
+module sts_roles {
+    source  = "rh-mobb/rosa-sts-roles/aws"
```

Comment:
This module doesn't support a dynamic operator role names... 
I write it from scratch - here is the module https://github.com/terraform-redhat/terraform-provider-ocm/tree/main/modules/aws_roles and it contains one template for all the operator roles (in order to support dynamic names) https://github.com/terraform-redhat/terraform-provider-ocm/blob/main/modules/aws_roles/operator_roles/operator_role_resource.tf 


I'm working on moving it out into a new repository 

Please change it to be like in that example - https://github.com/terraform-redhat/terraform-provider-ocm/blob/main/examples/create_rosa_cluster/create_rosa_sts_cluster/operator_roles_and_oidc/main.tf#L41
`    source  = "git::https://github.com/terraform-redhat/terraform-provider-ocm.git//modules/aws_roles"
`
And when I will move it out into a new repository I'll change it here also